workspaces/arborist/test/override-set.js

```diff
@@ -271,4 +271,121 @@ t.test('constructor', async (t) => {
     const outOfRangeRule = bazEdgeRule.getEdgeRule({ name: 'buzz', spec: 'github:baz/buzz#semver:^2.0.0' })
     t.equal(outOfRangeRule.name, 'baz', 'no match - returned parent')
   })
+
+  t.test('isequal and findspecificoverrideset tests', async (t) => {
```

Comment:
Can you add comments to the test explaining the scenario you're testing. This would be helpful for others digging into it later.